dgen_os/python/input_data_functions.py

```diff
@@ -524,50 +579,50 @@ def process_wholesale_elec_prices(wholesale_elec_price_traj):
 
 
 #%%
-def process_load_growth(load_growth):
-    """
-    Returns the trajectory of the load growth rates over time relative to a base year of 2014
+# def process_load_growth(load_growth):
```

Comment:
As above, can remove this chunk if it is unused/outdated as the code will be tracked through GitHub